projects/packages/videopress/src/class-admin-ui.php

```diff
@@ -124,18 +143,19 @@ public static function render_initial_state() {
 	 */
 	public static function initial_state() {
 		return array(
-			'apiRoot'           => esc_url_raw( rest_url() ),
-			'apiNonce'          => wp_create_nonce( 'wp_rest' ),
-			'registrationNonce' => wp_create_nonce( 'jetpack-registration-nonce' ),
-			'adminUrl'          => self::get_admin_page_url(),
-			'adminUri'          => 'admin.php?page=' . self::ADMIN_PAGE_SLUG,
-			'paidFeatures'      => array(
+			'apiRoot'                => esc_url_raw( rest_url() ),
+			'apiNonce'               => wp_create_nonce( 'wp_rest' ),
+			'registrationNonce'      => wp_create_nonce( 'jetpack-registration-nonce' ),
+			'adminUrl'               => self::get_admin_page_url(),
+			'adminUri'               => 'admin.php?page=' . self::ADMIN_PAGE_SLUG,
+			'paidFeatures'           => array(
 				'isVideoPressSupported'          => Current_Plan::supports( 'videopress' ),
 				'isVideoPress1TBSupported'       => Current_Plan::supports( 'videopress-1tb-storage' ),
 				'isVideoPressUnlimitedSupported' => Current_Plan::supports( 'videopress-unlimited-storage' ),
 			),
-			'productData'       => My_Jetpack_Products::get_product( 'videopress' ),
-			'siteSuffix'        => ( new Status() )->get_site_suffix(),
+			'productData'            => My_Jetpack_Products::get_product( 'videopress' ),
+			'siteSuffix'             => ( new Status() )->get_site_suffix(),
+			'allowedVideoExtensions' => self::get_allowed_video_extensions(),
```

Comment:
can we expose a `allowedVideoMimeTypes` or something like that too?

Reply:
I have two ideas on how to do it:

* having a separated list of `allowedVideoMimeTypes`
* returning it after each extension, sort of like the `get_allowed_mime_types()` functions does, like this:

```
...
"allowedVideoExtensions" => [
    "asf" => "video/x-ms-asf",
    "asx" => "video/x-ms-asf", // repeated mime type, just to be associated with each possible extension
    "wmv" => "video/x-ms-wmv",
    ...
],  
```

Which one do you think is better?

Reply:
second one. we are plenty of excellent native methods to process the data on the client side to get what we need.

Reply:
Done, now the returned value is as follow:

```
"allowedVideoExtensions" => [
    "asf" => "video/x-ms-asf",
    "asx" => "video/x-ms-asf",
    "wmv" => "video/x-ms-wmv",
    "wmx" => "video/x-ms-wmx",
    "wm" => "video/x-ms-wm",
    "avi" => "video/avi",
    "divx" => "video/divx",
    "flv" => "video/x-flv",
    "mov" => "video/quicktime",
    "qt" => "video/quicktime",
    "mpeg" => "video/mpeg",
    "mpg" => "video/mpeg",
    "mpe" => "video/mpeg",
    "mp4" => "video/mp4",
    "m4v" => "video/mp4",
    "ogv" => "video/ogg",
    "webm" => "video/webm",
    "mkv" => "video/x-matroska",
    "3gp" => "video/3gpp",
    "3gpp" => "video/3gpp",
    "3g2" => "video/3gpp2",
    "3gp2" => "video/3gpp2",
    "videopress" => "video/videopress",
],
```

Thank you for the suggestion!